examples/astro-forms-demo/src/styles/global.css

```diff
@@ -0,0 +1,12 @@
+/*
+
+██╗    ██╗ █████╗ ██████╗ ███╗   ██╗██╗███╗   ██╗ ██████╗
+██║    ██║██╔══██╗██╔══██╗████╗  ██║██║████╗  ██║██╔════╝
+██║ █╗ ██║███████║██████╔╝██╔██╗ ██║██║██╔██╗ ██║██║  ███╗
+██║███╗██║██╔══██║██╔══██╗██║╚██╗██║██║██║╚██╗██║██║   ██║
+╚███╔███╔╝██║  ██║██║  ██║██║ ╚████║██║██║ ╚████║╚██████╔╝
+ ╚══╝╚══╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═══╝╚═╝╚═╝  ╚═══╝ ╚═════╝
+
+Do not put here anything, refer to the classes available at
+@wix/wix-vibe-plugins/plugins-theme.css
```

Comment:
It is OK to use these in the example but in the plugin files don't use this css and use the vibe theme classes directly (like `text-primary-foreground`)